odoo/modules/module_graph.py

```diff
@@ -143,7 +143,10 @@ def __init__(self, name: str, module_graph: ModuleGraph) -> None:
         self.name: str = name
         # for performance reasons, use the cached value to avoid deepcopy; it is
         # acceptable in this context since we don't modify it
-        self.manifest: dict = _get_manifest_cached(name)
+        manifest = Manifest.for_addon(name)
```

Comment:
@kmagusiak this seems to log a warning when starting a db with imported modules (manifest not found). Do you mind having a look?

Reply:
#216118